compiler-rt/lib/scudo/standalone/tsd_shared.h

```diff
@@ -93,13 +93,15 @@ struct TSDRegistrySharedT {
 
   void disable() NO_THREAD_SAFETY_ANALYSIS {
     Mutex.lock();
+    MutexTSDs.lock();
```

Comment:
Can we also add ACQUIRED_BEFORE(MutexTSDs) on Mutex? Even though there's no thread safety analysis for enable()/disable() but I guess in init() this can be useful.